packages/perseus/src/widgets/interactive-graphs/stateful-mafs-graph.tsx

```diff
@@ -116,12 +116,19 @@ export const StatefulMafsGraph = React.forwardRef<
     const originalPropsRef = useRef(props);
     const latestPropsRef = useLatestRef(props);
     useEffect(() => {
+        if (state.hasBeenInteractedWith) {
+            return;
+        }
```

Comment:
I think this change might cause problems for graph previews in the content editor. Once you interact with the preview, it will never reflect changes to the graph settings (`numPoints`, `numSegments`, etc.) What's the intent of this change? Is there another way it could be accomplished?

Reply:
I would love to work through some way to make this work. When reinitialize is called it wipes away the flag to show the keyboard invitation. The re-initialization is getting called as soon as the graph has focus and so the prompt doesn't get seen until the second time.